src/api/routes.py

```diff
@@ -0,0 +1,11 @@
+"""'api' package's routes module."""
+
+from flask import Blueprint
+from flask_restx import Api
+
+from src.api.users.routes import api as users
+
+
+bp = Blueprint("api", __name__)
+api = Api(bp)
+api.add_namespace(users, path="/api")
```

Comment:
Seems like prefix "/api" should be set not for the users namespace but for the whole api Blueprint. And users Namespace in turn should have "/users" prefix. 

After we add "/users" prefix to our users Namespace, we will be able to remove "/users" parts from routes in `src/api/users/routes.py`